.github/workflows/documentation.yml

```diff
@@ -0,0 +1,48 @@
+name: Documentation
+
+on:
+  push:
+    branches: main
+    tags: '*'
+  workflow_dispatch:  # Allow manually triggering the workflow
+
+concurrency:
+  group: >-
+    ${{ github.workflow }}-${{ github.ref_type }}-
+    ${{ github.event.pull_request.number || github.sha }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pages: write
+      id-token: write
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with: {python-version: "3.11"}
+      - name: Build Docs
+        run: |
+          pip install tox
+          tox -e docs-py311
```

Comment:
one more thought we need to consider (not necessariyl this PR but related to docs building): Is is reasonable to require the functionality of ALL links? 

This 
1) blocks us from releasing the doc for the first time (since theres no website there yet and many links are broken but if the pipeline that builds these links also requires them its a ciruclar problem)
2) Blocks us everytime we adjust the webpage with sites that are not there yet on the released version but are references in the new version

Is there a way to exlcude the doc pages themselves from the link checker?

Reply:
It is possible to tell it ignore certain links by giving it a regex (see [here](https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-the-linkcheck-builder) for more options):
<img width="845" alt="image" src="https://github.com/emdgroup/BayBE/assets/84016733/4a3eea8a-294a-4fb8-b8ea-3b4ff33ce794">
We could thus just add a list of these into our `conf.py` file. If that makes sense, I could already implement that in the furo theme PR.

Reply:
If I should include that, please just provide me a list/regex here and feel free to resolve.

Reply:
I now included a corresponding check in the furo_theme PR and am hence resolving this.